packages/react/package.json

```diff
@@ -19,9 +19,9 @@
     "prerelease": "gitpkg publish"
   },
   "dependencies": {
-    "@gadgetinc/api-client-core": "^0.13.10",
+    "@gadgetinc/api-client-core": "^0.13.14",
```

Comment:
Our react bindings now need the newest api-client-core which also depends on urql 4, as well as depending on urql 4 themselves